<launch>
	<!-- IMU NODE -->
	<node pkg="barc" type="imu_data_acquisition.py" name="imu_node" >
		<param name="port" value="/dev/ttyACM0" />
	</node>

	<!-- DATA SERVICE NODE -->
	<!-- <node pkg="data_service" type="service.py" name="data_service_node" > -->
	<!-- </node> -->
</launch>
